lib/YaST/Module.pm

```diff
@@ -0,0 +1,43 @@
+# SUSE's openQA tests
+#
+# Copyright © 2021 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+
+# Summary: The class introduces methods that are common for all YaST modules.
+# Maintainer: QE YaST <qa-sle-yast@suse.de>
+
+package YaST::Module;
+use strict;
+use warnings;
+use testapi;
+use YuiRestClient;
+use y2_module_guitest qw(launch_yast2_module_x11);
+use y2_module_consoletest qw(yast2_console_exec);
+
+sub open {
+    my ($args) = @_;
+    my $module = $args->{module};
+    my $ui     = $args->{ui};
+    die 'No module name specified.'    unless defined $module;
+    die 'No user interface specified.' unless defined $ui;
+    if (lc($ui) eq 'ncurses') {
+        y2_module_consoletest::yast2_console_exec(
```

Comment:
As here we do not inherit from `y2_module_guitest`, we can get rid of `y2_module_consoletest::`